systemd/schema-registry.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Schema Registry
+Documentation=http://docs.confluent.io/current/schema-registry/docs/index.html
+Requires=network.target
+Wants=kafka-server.service
```

Comment:
We don't have `kafka-server.service` yet, right? So if we merge this now, will it even run with this script?